hbw/app/assets/javascripts/hbw_dependencies/noconflict.js.coffee

```diff
@@ -9,12 +9,12 @@
 #= require react
 #= require ./wrap_globals
 
-HBWDependencies = ['$', 'jQuery', 'moment', 'React']
+HBWDependencies = ['$', 'jQuery', 'moment', 'React', 'define']
```

Comment:
Yes, that is what my changes are all about. http://stackoverflow.com/questions/24746175/how-can-i-disable-require-js-at-runtime